### DISCUSSION TITLE
是否考虑将 CKylinMC的关注管理器 加入 Bilibili-Evolved

### DISCUSSION THREAD

**Discussion by ShoiLyaung** [the1812/Bilibili-Evolved#4966] [功能建议 / Ideas]
- [CKylinMC的关注管理器](https://greasyfork.org/en/scripts/428895-bilibili-%E5%85%B3%E6%B3%A8%E7%AE%A1%E7%90%86%E5%99%A8) 的功能非常实用
- 是否考虑将其他 [CKylinMC的哔哩哔哩脚本](https://greasyfork.org/zh-CN/scripts?language=all&set=403506) 加入仓库，这些脚本提供了非常实用功能。

**Comment by WakelessSloth56** [comment]
它们是 `GPLv3` 许可的，直接加进来肯定不行的。

不过原作者说是

> 抱歉，因工作问题暂停绝大部分脚本更新。

你可以去找它的作者商讨一下？

---

或者可以基于它做成以 GPL 许可的第三方组件，然后添加进来。

但是目前 Bilibili-Evolved 只能添加复刻的同名仓库中的第三方组件，还是不太好处理许可问题， @the1812 能否支持添加任意 URL 的第三方组件到 `third-party.ts` ？

---

净室设计一组功能相同的组件也是一种方法，但是就……不太合适吧（

**Comment by the1812** [comment]
那为什么不直接用这个脚本呢, 强行加进来没有什么意义
